### ISSUE TITLE
i18n-named-placeholders rule does not handle empty translate calls, crashes eslint

### ISSUE THREAD

**Issue by deBhal** [Automattic/eslint-plugin-wpcalypso#43]
If a document contains `translate()` the `i18n-named-placeholders` rule throws a `TypeError`, crashing `eslint`:

```
Cannot read property 'type' of undefined
TypeError: Cannot read property 'type' of undefined
    at getTextContentFromNode ([..]/eslint-plugin-wpcalypso/lib/util/get-text-content-from-node.js:32:11)
    at EventEmitter.CallExpression ([..]/eslint-plugin-wpcalypso/lib/rules/i18n-named-placeholders.js:53:15)
[...]
```

**Comment by aduth** [comment]
This should have been fixed by #42, included in 3.4.1 release. Can you check that you're running the latest version?

**Comment by deBhal** [comment]
> This should have been fixed by #42, included in 3.4.1 release. Can you check that you're running the latest version?

👍  Sorry for the dupe.
